### ISSUE TITLE
categorial y axis

### ISSUE THREAD

**Issue by smsaladi** [eclarke/ggbeeswarm#3]
Thanks for bringing beeswarm to ggplot. It's really very useful.

Would it be possible to add functionality for categorial values on the y-axis? Right now, I need to do the following to get it to work as I'd like:

```{R}
ggplot(df_outcomes, aes(x=factor(outcome), y=ml21)) + 
     geom_jitter(position = position_quasirandom()) + 
     coord_flip()
```

position_jitter can handle this by specifying the direction to jitter on:

```{R}
ggplot(df_outcomes, aes(x=ml21, y=factor(outcome))) + 
     geom_jitter(position = position_jitter(height = 0.2))
```

This would be super useful to me (and perhaps others) -- not having to do the coord_flip would make the other geoms in the same plot easier to deal with/whole plotting more streamlined/easier to communicate & understand.

Many thanks.

**Comment by smsaladi** [comment]
It would also make it possible to use facet_wrap (which doesn't support coord_flip) in conjunction with ggbeeswarm with categorial y axes. [a problem that I just ran up against]

**Comment by smsaladi** [comment]
I actually needed this functionality sooner than later, so I implemented it myself. I just submitted pull request for your review.

**Comment by eclarke** [comment]
Great, thank you! Merged in #5.
